filament/src/PostProcessManager.cpp

```diff
@@ -590,7 +636,7 @@ PostProcessManager::StructurePassOutput PostProcessManager::structure(FrameGraph
                 FMaterial const* const ma = material.getMaterial(mEngine);
                 auto pipeline = getPipelineState(ma);
 
-                FMaterialInstance* const mi = PostProcessMaterial::getMaterialInstance(ma);
+                FMaterialInstance* const mi = mMaterialInstanceManager.getMaterialInstance(ma);
```

Comment:
For all these you could maybe have a helper on PostProcessManager so you don't have to repeat `mMaterialInstanceManager.` each time.

Reply:
done